### ISSUE TITLE
Feature request: copy to clipboard on vim keybindings with ctrl + c

### ISSUE THREAD

**Issue by michelececcacci** [Anderbone/leetcode-shortcut-browser-extension#3]
When using vim keybidings, copying text doesn't  actually copy in the  clipboard and it's very inconvenient. Since i feel it's similar to what this extension is trying to achieve (and it can be useful) I can take a look and try to implement it

**Comment by Anderbone** [comment]
> When using vim keybidings, copying text doesn't actually copy in the clipboard and it's very inconvenient. Since i feel it's similar to what this extension is trying to achieve (and it can be useful) I can take a look and try to implement it

Hi thank you for your suggestions, I'm not sure if this one helps: https://chowdera.com/2022/02/202202070903192451.html
Yeah give it a try if you want to implement it, thanks.

**Comment by michelececcacci** [comment]
I can try in the next days. Also, another thing, are you open to migrating to typescript instead of JS? i feel it could reduce problems when the code becomes more complex

**Comment by Anderbone** [comment]
Yes that's a good idea, but I may not have any time in next week and I don't know typescript yet. But this is a so small project so welcome to do the changes if you have time.

**Comment by michelececcacci** [comment]
Ok, i'm not really good at typescript either, but i worked with statically typed languages before, so i think i can make it work!
